tests/tensorflow/quantization/test_algorithm_quantization.py

```diff
@@ -278,6 +278,7 @@ def get_quantize_inputs_test_model(input_shapes):
 
 def test_quantize_inputs():
     config = get_basic_quantization_config()
+    config['target_device'] = 'TRIAL'
```

Comment:
@vshampor wrote above

> This test case should either use a model where the inputs are not unified by scales, or a 'TRIAL' mode - in essence, an option prohibiting the unified scale invocation - so that the test case is more limited in what it actually requires from NNCF in order to pass.